rh/schema.py

```diff
@@ -26,6 +37,34 @@ def get_slack_notify_text(rhr: models.RentalHistoryRequest) -> str:
     return f"{user_text} has requested {rh_link}!"
 
 
+def run_rent_stab_sql_query(bbl: str) -> Optional[Dict[str, Any]]:
+    sql_query = """
+        select uc2007, uc2008, uc2009, uc2010, uc2011, uc2012, uc2013,
+             uc2014, uc2015, uc2016, uc2017, uc2018, uc2019
+        from rentstab
+        full join rentstab_v2 using(ucbbl)
+        where ucbbl = %(bbl)s
+    """
+    if not settings.NYCDB_DATABASE:
+        return None
+
+    with connections[settings.NYCDB_DATABASE].cursor() as cursor:
+        cursor.execute(sql_query, {"bbl": bbl})
+        json_result = list(generate_json_rows(cursor))
+        if not json_result:
+            return BLANK_RENT_STAB_INFO
+        return json_result[0]
```

Comment:
Ideally I think it'd be good if, in the future, we could leverage the `nycdb` python package to scaffold some of this stuff for us, so that we could actually test this code.  That's beyond the scope of this PR but it'd be good to file an issue reminding us to do it later...